### PR TITLE
Allow Client 10 (client-10/12.x branch)

### PR DESCRIPTION
**Pull request by hyresu**
### DISCLAIMER:
**Don't use this pull request if you are not using the outdated-10/12.x branch, which has been created on 12 January after #2265. You may encounter bugs, warnings and crashs if you are using develop branch and add these changes. 
THIS PULL REQUEST IS OPTIONAL AS WILL ONLY ADD CLIENT 10 SUPPORT, IF YOU WISH TO USE ONLY CLIENT 12 YOU DON'T NEED TO ADD THESE CHANGES** 

[UP TO DATE WITH #2437]

TODO:

- [x] Store (Just the base)
- [x] Loot message
- [x] Loot channel

### CODE REVIEW

**Review by aurelion5670 — APPROVED** [review]

**Comment by Bipbop69** [comment]
testing I had debug when logging 10/12.60.
 
edit
in client 12.60  login ok  
client 10 login ok
-> debug outfit

**Comment by Bipbop69** [comment]
clicking on a store subcategory debug client 10 occurs

**Comment by luucassoares** [comment]
Hello.. I have a doubt... would this work with https://github.com/OTCv8/otclientv8 client?

**Comment by hyresu** [comment]
> clicking on a store subcategory debug client 10 occurs

Only in consumables, right?

> Hello.. I have a doubt... would this work with https://github.com/OTCv8/otclientv8 client?

Yes

**Comment by Bipbop69** [comment]
> > clicking on a store subcategory debug client 10 occurs
> 
> Only in consumables, right?
> 
> > Hello.. I have a doubt... would this work with https://github.com/OTCv8/otclientv8 client?
> 
> Yes

all that have subcategory

**Comment by antonioaraujo664** [comment]
Is there any way to find out which player's client at login?

File: src/protocolgame.cpp [hunk 1]
```diff
@@ -2987,12 +3010,19 @@ void ProtocolGame::sendBlessStatus()
 	}
 
 	msg.addByte(0x9C);
+	if (version > 1200) {
```
Comment (karimadnan) [inline src/protocolgame.cpp]: if (version >= 1200) 
you forgot =

Comment (hyresu) [reply src/protocolgame.cpp]: Thank you!

Comment (karimadnan) [reply src/protocolgame.cpp]: do you have discord?

Comment (dex-89) [reply src/protocolgame.cpp]: Sure, he is at discord https://discordapp.com/invite/3NxYnyV

**Comment by methx** [comment]
Encontré un nuevo bug, si tienes activada alguna funcion del cliente 12, por ejemplo el track del besteary y logeas e intentas entrar en el OTC no podras conectar y si usas el tibia 10, te dara debug, necesitas volver a entrar y desactivar el track del besteary para poder utilizar el otc o el cliente 10

**Review by majestyotbr — APPROVED** [review]

**Pull request merged**